src/Plugin/VoteUpDownWidgetInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Vote Up/Down Widget plugins.
+ */
+interface VoteUpDownWidgetInterface extends PluginInspectionInterface {
```

Comment:
Please add the two missing methods to the interface. I [mentioned this before](https://github.com/marvil07/vote_up_down/pull/3#discussion_r122375874):

> ```
> # On D7
> git grep ctools_plugin_get_function
> vud.theme.inc:  if ($function = ctools_plugin_get_function($plugin, 'alter template variables')) {
> vud.theme.inc:    if ($function = ctools_plugin_get_function($plugin, 'ajax render')) {
> vud.theme.inc:      if ($function = ctools_plugin_get_function($plugin, 'ajax render')) {
> 
> ```
> i.e. a couple related methods missing on the interface.